ui/src/shared/components/ThresholdMarkers.tsx

```diff
@@ -36,18 +36,6 @@ const ThresholdMarkers: FunctionComponent<Props> = ({
       [field]: yValue,
     }
 
-    if (
```

Comment:
I think removing this will mean that when a user drags the max below the min, then the threshold type will suddenly change from `within: true` to `within: false`, which sounds really jarring.

I imagine a way we can handle this is by determining the `within` value of a threshold when it is first instantiated, storing that in state, then treating the threshold accordingly. But that sounds complicated. I would really love to get those old `Threshold` types back.

Reply:
https://github.com/influxdata/influxdb/issues/14537